startup/gui/ocio.py

```diff
@@ -78,21 +78,24 @@
 
 def __setDisplayTransform() :
 
-	d = OCIO.DisplayTransform()
-	d.setInputColorSpaceName( OCIO.Constants.ROLE_SCENE_LINEAR )
+	d = OCIO.DisplayViewTransform()
+	d.setSrc( OCIO.ROLE_SCENE_LINEAR )
 	d.setDisplay( defaultDisplay )
 	d.setView( preferences["displayColorSpace"]["view"].getValue() )
 
-	context = config.getCurrentContext().createEditableCopy()
+	# \todo : Should be `context = copy.deepcopy( config.getCurrentContext() )`
+	# once https://github.com/AcademySoftwareFoundation/OpenColorIO/pull/1575 gets merged into OCIO2.1.2
+	context = OCIO.Context( searchPaths = list( config.getCurrentContext().getSearchPaths() ), workingDir = config.getCurrentContext().getWorkingDir(), environmentMode = config.getCurrentContext().getEnvironmentMode(), stringVars = dict( config.getCurrentContext().getStringVars() ) )
 	for variable in preferences["displayColorSpace"]["context"] :
 		if variable["enabled"].getValue() :
-			context.setStringVar( variable["name"].getValue(), variable["value"].getValue() )
+			context[ variable["name"].getValue() ] = variable["value"].getValue()
 
 	processor = config.getProcessor( d, context = context )
```

Comment:
I needed `config.getProcessor( transform = d, context = context, direction = OCIO.TRANSFORM_DIR_FORWARD )` here, otherwise PyBind told me I had the wrong arguments.